style.css

```diff
@@ -0,0 +1,80 @@
+html {
+    height: 100vh;
+}
+body {
+    font-family: arial, sans-serif;
+}
+
+form {
+    height: 100%;
+    width: 100%;
+}
+
+/* Grid layout */
+.main-container {
+    display: grid;
+    height: 97vh;
+    width: 100%;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-template-areas:
+    "search-container new-task-container"
+    "tasks-container tasks-container";
+    border: 1px black solid;
+    border-radius: 3px;
+}
+
+.search-container {
+    grid-area: search-container;
+    border: 1px black solid;
```

Comment:
create a `.box` class (or any other name you'd like) for this border styling, and use it on the different elements on the page